CMakeLists.txt

```diff
@@ -408,8 +408,14 @@ include(px4_add_gtest)
 if(BUILD_TESTING)
 	include(gtest)
 
+	set(TESTFILTERARG "")
```

Comment:
@mrpollo May I ask why this was changed? I breaks the unit test filter because it ignores the argument and just always leaves it empty. Can we revert?

Reply:
For some reason something within gtest/ctest was failing when running `make tests`.  The failure was something like "Unknown -R argument."

What we did here is to avoid calling the test without `-R` when there's no argument to filter through.

Reply:
Not sure what you mean but this change just breaks the filter. Now when working on any branch from latest main I have to wait for all the tests for every single development iteration 👀 Should I open a pr to revert?

Reply:
https://github.com/PX4/PX4-Autopilot/pull/23978 😅